### PR TITLE
Create de.json

### PR DESCRIPTION
**Pull request by dirkg173**
Could you please check if this is correct?

### CODE REVIEW

**Comment by runely** [comment]
This seems correct. However, there are a few more files that also needs to be translated into German. The full list of files is commented [here](https://github.com/runely/calendar-homey/issues/62#issuecomment-671097681). Update this pull request with the rest of the files or a new pull request with the rest of the files. 👍

File: locales/de.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "conditions_event_name_start_stop_date_year_format": "MM.DD.YY",
+    "token_duration_minutes": "Minuten",
+    "token_duration_hour": "Stunde",
+    "token_duration_hours": "Stunden",
+    "calendarTokens": {
+        "events_today_calendar_title_stamps": "Heutige Ereignisse in",
+        "events_tomorrow_calendar_title_stamps": "Morgige Ereignisse in"
+    },
+    "flowTokens": {
+        "event_next_title": "Nächster Veranstaltungstitel",
```
Comment (dirkg173) [inline locales/de.json:29]: Please change in: "Nächster Ereignistitel"

File: locales/de.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_startstamp": "Beginn der nächsten Veranstaltung",
```
Comment (dirkg173) [inline locales/de.json:30]: Please change in: "Beginn des nächsten Ereignisses"

File: locales/de.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_startstamp_date_time_format": "MM.DD HH:mm",
+        "event_next_startstamp_date_format": "MM.DD",
+        "event_next_stopstamp": "Ende der nächsten Veranstaltung",
+        "event_next_stopstamp_date_time_format": "MM.DD HH:mm",
+        "event_next_stopstamp_date_format": "MM.DD",
+        "event_next_duration": "Nächste Veranstaltungsdauer",
```
Comment (dirkg173) [inline locales/de.json:36]: Please change in: "Nächste Ereignisdauer"

File: locales/de.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_duration_minutes": "Nächste Veranstaltungsdauer (Minuten)",
```
Comment (dirkg173) [inline locales/de.json:37]: Please change in: "Nächste Ereignisdauer (Minuten)"

File: locales/de.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_starts_in_minutes": "Nächste Veranstaltung beginnt in (Minten)",
```
Comment (dirkg173) [inline locales/de.json:38]: Please change in: "Nächstes Ereignis beginnt in (Minuten)"

File: locales/de.json [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_stops_in_minutes": "Nächste Veranstaltung endet in (Minuten)",
```
Comment (dirkg173) [inline locales/de.json:39]: Please change in: "Nächstes Ereignis endet in (Minuten)"

File: locales/de.json [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+        "event_next_calendar_name": "Kalendername der nächsten Veranstaltung",
```
Comment (dirkg173) [inline locales/de.json:40]: Please change in: "Kalenderbezeichnung des nächsten Ereignisses"

File: locales/de.json [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    "calendarTokens": {
+        "events_today_calendar_title_stamps": "Heutige Ereignisse in",
+        "events_tomorrow_calendar_title_stamps": "Morgige Ereignisse in"
+    },
+    "flowTokens": {
+        "event_next_title": "Nächster Veranstaltungstitel",
+        "event_next_startstamp": "Beginn der nächsten Veranstaltung",
+        "event_next_startstamp_date_time_format": "MM.DD HH:mm",
+        "event_next_startstamp_date_format": "MM.DD",
+        "event_next_stopstamp": "Ende der nächsten Veranstaltung",
```
Comment (dirkg173) [inline locales/de.json:33]: Please change in: "Ende des nächsten Ereignisses"

File: locales/de.json [hunk 9]
```diff
@@ -0,0 +1,51 @@
+{
+    "settings": {
+        "title": "Einstellungen",
+        "legend": "URI zur .ics-Datei",
+        "calendar_name": "Kalendername",
```
Comment (dirkg173) [inline locales/de.json:5]: Please change in: "Kalenderbezeichnung"

File: locales/de.json (hunk 1 above)
Comment (runely) [reply locales/de.json:29]: Done

File: locales/de.json (hunk 2 above)
Comment (runely) [reply locales/de.json:30]: Done

File: locales/de.json (hunk 3 above)
Comment (runely) [reply locales/de.json:36]: Done

File: locales/de.json (hunk 4 above)
Comment (runely) [reply locales/de.json:37]: Done

File: locales/de.json (hunk 5 above)
Comment (runely) [reply locales/de.json:38]: Done

File: locales/de.json (hunk 6 above)
Comment (runely) [reply locales/de.json:39]: Done

File: locales/de.json (hunk 7 above)
Comment (runely) [reply locales/de.json:40]: Done

File: locales/de.json (hunk 8 above)
Comment (runely) [reply locales/de.json:33]: Done

File: locales/de.json (hunk 9 above)
Comment (runely) [reply locales/de.json:5]: DOne

**Pull request merged**